### DISCUSSION TITLE
支持twikoo、artalk第三方评论插件的内容导入

### DISCUSSION THREAD

**Discussion by carolcoral** [halo-dev/halo#4788] [Ideas]
### Your current Halo version

2.10.0

### Describe this feature

因为原来的halo自带的评论插件不够完善，一直使用的twikoo评论服务。
目前想要迁移回官方评论插件，能否提供插件或功能用于迁移twikoo/artalk第三方评论服务的内容到官方评论中

**Comment by ruibaby** [comment]
Halo 官方可能没有精力做这样的插件或者功能，看看社区有没有人有兴趣做吧。
